src/core/openthread-core-default-config.h

```diff
@@ -186,6 +186,17 @@
 #endif  // OPENTHREAD_CONFIG_MAX_JOINER_ENTRIES
 
 /**
+ * @def OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS
+ *
+ * The number of message buffers in the buffer pool.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_MAX_STATECHANGE_HANDLERS
+#define OPENTHREAD_CONFIG_MAX_STATECHANGE_HANDLERS          2
```

Comment:
Maybe default this to 1, and only when there are platforms that need multiple do they set it to something else?